src/frontend/Komponenter/Behandling/VedtakOgBeregning/Felles/RevurderesFraOgMed.tsx

```diff
@@ -19,11 +19,13 @@ const Advarsel = styled(AlertWarning)`
 type Type = 'OVERGANGSSTØNAD' | 'BARNETILSYN';
 
 const revurderesFørFørstePeriodeAdvarsel = (type: Type) => {
```

Comment:
Burde vi endre navn på 
 * revurderesFørFørstePeriodeAdvarsel
 * revurdererFørFørstePeriode

Og i så fall, forslag?

Reply:
`revurdersFraPeriodeUtenTidligereStønadAdvarsel` 😂

Reply:
Eirik foreslo `revurdererFraPeriodeUtenStønad` på slack så gikk for det :D